frontend_lib/src/component/Timeline/DisplayAddedFileToComment.styl

```diff
@@ -0,0 +1,44 @@
+@import '../../css/Variable.styl'
+
+.DisplayAddedFileToComment
+  color fontColor
+  &__message
+    display flex
+    align-items center
+    width max-content
+    padding-inline-start 8px
+    cursor pointer
+    font-size fontSizeSmall
+    & > &__iconFile, & > &__text
+      margin-inline-end 5px
+    & > &__iconFile, & > &__listIcon
+      font-size 16px
+    &__listIcon
+      color mediumDarkGrey
+    &:hover
+      & .DisplayAddedFileToComment__message__listIcon
+        color darkGrey
+        text-decoration underline
+  &__popover__item
+    display flex
+    align-items center
+    padding-top 8px
+    padding-bottom 8px
+    padding-inline 12px
+    &:not(:last-child)
+      border-bottom 1px solid mediumDarkGrey
```

Comment:
Almost all borders at Tracim has `grey` color, is there a reason for this one to be `mediumDarkGrey`?

Reply:
mmh I don't think there is
will fix